pkg/policies/controlplane/runtime/tristate/tristate.go

```diff
@@ -63,6 +63,21 @@ func FromReading(reading runtime.Reading) Bool {
 	return True
 }
 
+// IsTrue returns the tri-state boolean True.
+func IsTrue() Bool {
```

Comment:
These methods don't make much sense. What Michal was saying that add helper methods for Bool Type. e.g. func (bool Bool) IsTrue()

Reply:
Done. I also added isFalse and isUnkown because if it is not true we have to check if it is either false or unknown.